src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -46,7 +46,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _lastMouseClickPos{},
         _selectionNeedsToBeCopied{ false }
     {
-        _id = ControlInteractivity::_nextId++;
+        _id = _nextId.fetch_add(1, std::memory_order_relaxed);

```

Comment:
Editor's note: you can have up to 18,446,744,073,709,551,615 terminal panes in the same process before this becomes a problem; it is likely that we will crash before you get to there :)